openshift/operator-controller.Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20 AS builder
+
+ARG SOURCE_GIT_COMMIT
+ENV GIT_COMMIT=${SOURCE_GIT_COMMIT}
 WORKDIR /build
 COPY . .
-RUN make go-build-local && \
+RUN make -f openshift/Makefile go-build-local && \
```

Comment:
Why do we need to make this change to sort out the issue?

Reply:
Sorry, I should have removed this question. I put it first in the review, and then, after navigating, I got it.
Thank you a lot. 🙏